code/code/misc/statistics.cc

```diff
@@ -175,8 +175,16 @@ int init_game_stats(void)
 
     SetupStaticGoldModifiers();
 
-    if (fscanf(fp, "%f\n", &stats.equip) != 1) {
-      vlogf(LOG_BUG, "bad value for equipment load rate");
+    if (fscanf(fp, "%lf\n", &stats.equip) != 1) {
+      vlogf(LOG_BUG, "bad value for equipment load potential");
+    }
+
+    if (fscanf(fp, "%lf\n", &stats.global_lp_target) != 1) {
```

Comment:
Pretty sure this will break compatibility. New code won't be able to read old stats files. Why don't we put it into the DB, where this wouldn't be an issue, and moreover where we want to move things anyway?

Reply:
I'm not sure there's really a good way around this. I manually added the two numbers into the stats file to make this work. We could put them at the end if we wanted, would that be satisfactory? My preference is for keeping the related statistics together, which is why I did it the way I did.

Reply:
I just moved these values to the end of the file, so any old versions reading the file won't break anything.